scalafmt-core/shared/src/main/scala/org/scalafmt/config/ContinuationIndent.scala

```diff
@@ -4,18 +4,29 @@ import metaconfig._
 
 /**
   * @param defnSite indentation around class/def
+  * @param declSite indentation around type/method declarations
+  * @param ctorSite indentation around class constructor parameters
   * @param callSite indentation around function calls, etc.
   * @param extendSite indentation before `extends`
   * @param withSiteRelativeToExtends additional indentation before `with`
   */
 case class ContinuationIndent(
     callSite: Int = 2,
     defnSite: Int = 4,
+    declSite: Option[Int] = None,
+    ctorSite: Option[Int] = None,
```

Comment:
I agree this name is most consistent with the other options since it matches the scala.meta.Tree names. Still, I regret using "defnSite/declSie" instead of "definitionSite" and "declarationSite"

Reply:
we can deprecate `defnSite` or use annotation that `metaconfig` allows for aliasing. or leave it be. what would you do?

Reply:
We can leave this unchanged for now and then separately give readable names to all options. We can continue to support the old names using metaconfig `@DeprecatedName(..)` https://github.com/scalameta/metaconfig/blob/aaab8feb0d7b22f6ae78dda3f291839537e2006b/metaconfig-core/shared/src/main/scala/metaconfig/annotation/Annotations.scala#L7